dynamo/tools/recipes.py

```diff
@@ -743,8 +745,8 @@ def velocity_N(
         "beta_k",
         "gamma_k",
     ]:
-        if i in var_columns:
-            del adata.var[i]
+        if i in vel_params_df.columns:
+            del vel_params_df[i]
 
```

Comment:
what  adata.uns["vel_params_names"]?

Reply:
The adata.uns["vel_params_names"] will be updated later with `update_vel_params(adata, params_df=vel_params_df)`.